app/Views/Templates/components/actions/dropdown.blade.php

```diff
@@ -0,0 +1,36 @@
+@props([
+    'buttonClass' => 'btn btn-primary',  // Default button class
+    'menuClass' => 'dropdown-content menu bg-base-100 rounded-box z-50 w-52 p-2 shadow',  // Default menu class
```

Comment:
That should not be a thing people can pass in. So just add it to the menu list

Reply:
i removed these 2 props